videointelligence/google/cloud/videointelligence_v1/gapic/video_intelligence_service_client.py

```diff
@@ -191,9 +191,9 @@ def __init__(
     # Service calls
     def annotate_video(
         self,
+        features,
```

Comment:
`features` was always required but was passed as a keyword arg. Since videointelligence is GA and this is a change to the v1, should I note this as a breaking change?

Reply:
Luke points out that `features` was always semantically required by the API. That backend would have thrown an error if it wasn't present.

Reply:
This is still technically a breaking change because it used to be an optional parameter in a different part of the signature.
E.g. the following would have been valid under the old signature:

```python
client.annotate_video(my_uri_str, features=list_of_features)
```

which would break under the new signature.

Reply:
I thought so as well, but it looks like you can continue to pass `features` as a keyword argument even though it's a positional parameter. 

https://medium.com/@boxed/keyword-argument-confusion-in-python-59105c5a1159

```py
def foo_old(a, b=None, c=None):
    print(a, b, c)

def foo_new(a, c, b=None):
    print(a, b, c)

foo_old(1, c=2, b=3)  # 1, 3, 2
foo_new(1, c=2, b=3)  # 1, 3, 2
```